### PR TITLE
winner winner chicken dinner with Julia

### PR DESCRIPTION
**Pull request by fruitl00p**
This woks using a clean Alpine image and a minor update on the raffler itself to make the randomness array value actually work...

### CODE REVIEW

File: duijf-julia/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM alpine:edge
+MAINTAINER robin@kingsquare.nl
+
+RUN apk add --no-cache julia --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
+
+# Create working dir
+RUN mkdir -p /var/app
+COPY ./raffler.jl /var/app
```
Comment (lucasvanlierop) [inline duijf-julia/Dockerfile]: I assume the newer docker versions actually support copying a file to a dir while the older versions expect the full path?

**Comment by lucasvanlierop** [comment]
I've updated Docker to 1.10, if you update with master you'll get the new version

File: duijf-julia/Dockerfile (hunk 1 above)
Comment (fruitl00p) [inline duijf-julia/Dockerfile]: thats my guess... hence my echo-ing of the docker version in travis... (they are running 1.8 currently)

**Comment by fruitl00p** [comment]
@lucasvanlierop it seems to fix it :) And so it is proven that even with docker the 'it works on my machine' still is a valid excuse.... :jack_o_lantern:

**Pull request merged**